build/Dockerfile

```diff
@@ -67,42 +67,36 @@ COPY /docker-entrypoint.sh /entrypoint.sh
 COPY /static $DSTA_HOME
 RUN mkfifo $DSTA_HOME/console \
 	&& chown -R $STEAM_USER:$STEAM_USER $DSTA_HOME \
-	&& mkdir -p `dirname $CONFIG_PATH` \
-	&& chown $STEAM_USER:$STEAM_USER `dirname $CONFIG_PATH`
+	&& mkdir -p `dirname $CLUSTER_PATH` \
+	&& chown $STEAM_USER:$STEAM_USER `dirname $CLUSTER_PATH`
 
 ARG SERVER_PORT=10999
 
 # Set environment variables.
-ENV SERVER_DESCRIPTION="Powered by DST Academy." \
+ENV DESCRIPTION="Powered by DST Academy." \
 	SERVER_PORT="$SERVER_PORT" \
-	OFFLINE_ENABLE=false \
 	MAX_PLAYERS=4 \
-	WHITELIST_SLOTS=0 \
```

Comment:
I'm going to put it back. We don't need it because the default value is 0. But if we add always all the settings it have more sense something like `whitelist_slots = 0` than `whitelist_slots = `. It's the same but it'd be more readable.